test/node_redis.spec.js

```diff
@@ -342,29 +342,6 @@ describe("The node_redis client", function () {
 
                 });
 
-                it('emits errors thrown from within an on("message") handler', function (done) {
-                    var client2 = redis.createClient.apply(redis.createClient, args);
-                    var name = 'channel';
-
-                    client2.subscribe(name, function () {
-                        client.publish(name, "some message");
-                    });
-
-                    client2.on("message", function (channel, data) {
-                        if (channel === name) {
-                            assert.equal(data, "some message");
-                            throw Error('forced exception');
```

Comment:
This code is written by the user and he should handle an error occuring in his code by himself without node_redis doing anything with that error.